src/assets/sheets_card.svg

```diff
@@ -0,0 +1 @@
+<svg width="558" height="300" fill="none" xmlns="http://www.w3.org/2000/svg"><g clip-path="url(#clip0)" fill-rule="evenodd" clip-rule="evenodd" fill="#2A2E45"><path d="M-61.525-63.513l-197.99 197.99a8 8 0 0 0 0 11.313l194.402 194.402a8 8 0 0 0 11.314 0l197.99-197.99a8 8 0 0 0 0-11.313L-50.211-63.513a8 8 0 0 0-11.314 0zm-199.404 196.575c-3.905 3.906-3.905 10.237 0 14.142l194.402 194.402c3.905 3.906 10.237 3.906 14.142 0l197.99-197.989c3.905-3.906 3.905-10.237 0-14.143L-48.797-64.928c-3.905-3.905-10.237-3.905-14.142 0l-197.99 197.99zM297.84 51.266L34.485 314.62a8 8 0 0 0 0 11.314l267.292 267.292a8 8 0 0 0 11.314 0l263.354-263.355a7.998 7.998 0 0 0 0-11.313L309.154 51.266a8 8 0 0 0-11.314 0zM33.071 313.206c-3.905 3.905-3.905 10.237 0 14.142L300.363 594.64c3.905 3.905 10.237 3.905 14.142 0L577.86 331.285c3.905-3.905 3.905-10.236 0-14.142L310.568 49.852c-3.906-3.906-10.237-3.906-14.142 0L33.071 313.205z"/><path d="M293.052-121.08l-400.149 400.149a8 8 0 0 0 0 11.314l272.38 272.38a8 8 0 0 0 11.313 0l400.15-400.15a8 8 0 0 0 0-11.313l-272.38-272.38a8 8 0 0 0-11.314 0zm-401.563 398.735c-3.905 3.905-3.905 10.237 0 14.142l272.38 272.38c3.905 3.905 10.236 3.905 14.142 0l400.149-400.15c3.905-3.905 3.905-10.236 0-14.142L305.78-122.494c-3.905-3.906-10.236-3.906-14.142 0l-400.149 400.149z"/><path d="M293.619 160.485L21.485 432.619a8 8 0 0 0 0 11.314l267.292 267.292a8 8 0 0 0 11.314 0l272.134-272.134a8 8 0 0 0 0-11.314L304.933 160.485a8 8 0 0 0-11.314 0zM20.071 431.205c-3.905 3.905-3.905 10.237 0 14.142l267.292 267.292c3.905 3.905 10.237 3.905 14.142 0l272.134-272.134c3.905-3.905 3.905-10.237 0-14.142L306.347 159.071c-3.905-3.905-10.237-3.905-14.142 0L20.071 431.205z"/><path d="M336.619 238.485L64.485 510.619a8 8 0 0 0 0 11.313l267.292 267.292a8 8 0 0 0 11.314 0L615.225 517.09a8 8 0 0 0 0-11.313L347.933 238.485a8 8 0 0 0-11.314 0zM63.071 509.205c-3.905 3.905-3.905 10.236 0 14.142l267.292 267.291c3.905 3.906 10.237 3.906 14.142 0l272.134-272.133c3.905-3.906 3.905-10.237 0-14.142L349.347 237.071c-3.905-3.906-10.237-3.906-14.142 0L63.071 509.205zM425.647-133.212L262.893 29.542a8 8 0 0 0 0 11.314l139.09 139.089a8 8 0 0 0 11.313 0L576.05 17.191a8 8 0 0 0 0-11.313l-139.089-139.09a8 8 0 0 0-11.314 0zM261.479 28.128c-3.905 3.905-3.905 10.237 0 14.142l139.089 139.089c3.906 3.905 10.237 3.905 14.143 0L577.464 18.606c3.905-3.906 3.905-10.237 0-14.142l-139.089-139.09c-3.905-3.905-10.237-3.905-14.142 0L261.479 28.128z"/><path d="M37.475 185.485l-197.99 197.99a8 8 0 0 0 0 11.314L33.887 589.191a8 8 0 0 0 11.314 0l197.99-197.99a8 8 0 0 0 0-11.314L48.789 185.485a8 8 0 0 0-11.314 0zm-199.404 196.576c-3.905 3.905-3.905 10.237 0 14.142L32.473 590.605c3.905 3.905 10.237 3.905 14.142 0l197.99-197.99c3.905-3.905 3.905-10.237 0-14.142L50.203 184.071c-3.905-3.905-10.237-3.905-14.142 0l-197.99 197.99z"/></g><defs><clipPath id="clip0"><path fill="#fff" d="M0 0h558v300H0z"/></clipPath></defs></svg>
```

Comment:
Why do we have this, if it is the same as for Sangraphs's card. @GarageInc

Reply:
yes, just use background from `dashboards` card from figma

Reply:
Fixed